### DISCUSSION TITLE
Weird "random" ordering

### DISCUSSION THREAD

**Discussion by jaredcwhite** [community/community#20191] [Other Feature Feedback, Questions, & Ideas] [answered]
I pray to god GitHub isn't testing some sort of feed sorting algorithm now. Because suddenly within the last few days my feed has changed from normal reverse chron to all over the map. One thing's from a few hours ago, then 5 days ago, then 2 days ago, then an hour ago. What the heck is happening?! Give me my reverse chron feed back please!!

**Comment by CHR-onicles** [comment]
Same!!

**Comment by belst** [comment]
I swapped back to `Following` Tab because it is completely unusable without chronological ordering for me.

**Comment by prathikshetty2002** [comment]
The color scheme of the github page also changed they are trying some new user experience methods to make users stay more on the github site seems like, but old is gold!

**Comment by aooohan** [comment]
Indeed, this is a very bad experience.

**Comment by siminapasat** [comment, marked as answer]
We experimented with a few algorithms (including a very non-ideal one, sorry @jaredcwhite you were in that very small user group), and based on the results [we reverted yesterday to a chronological feed](https://github.blog/changelog/2022-07-28-github-feed-public-beta-sorting-update/)

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
